packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -46,6 +52,10 @@ export class Snackbar implements ComponentInterface {
       this.handleSnackbarHide(e)
     );
     this.snackbarInstance.open();
+    this.snackbarInstance.timeoutMs = -1;
```

Comment:
I don't quite get the thoughts behind lines 55 to 57. Can't we just set the `this.snackbarInstance.timeoutMs` to `inoTimeout` and then call the `open` function? Otherwise, we rewrite the same logic of the MDC component, don't we?

Reply:
As I mentioned in the comment below, `setTimeout` only supports values in a specific range. Hence, I disabled mdc's timeout mechanism and added the `setTimeout` function to allow the consumer to set the property to any value.

Reply:
Alright, that's fine. 
One last thing: Coul you extract the timeout logic into its own function? For me, the `componentDidMount` hook is for setting up stuff like listeners and calling other functions, but not for the logic itself.

Reply:
Will do 🙂